src/SmartComponents/Inventory/applications/compliance/SystemPolicyCards.js

```diff
@@ -6,16 +6,27 @@ import propTypes from 'prop-types';
 import { Instagram } from 'react-content-loader';
 
 class SystemPolicyCards extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    systemPolicyCards() {
+        const policyCards = this.props.policies.map(
+            (policy, i) =>
+                <GridItem span={ 4 } key={ i }>
```

Comment:
Should this span `{ 4 }` on all screen sizes?

Reply:
Not really, I changed this to match the span of the cards in the main page of compliance